sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceActionFailureEventData.java

```diff
@@ -341,7 +341,7 @@ public String getClaims() {
      * @deprecated This method is no longer supported since v4.9.0.
      *     <p>Use {@link ResourceActionFailureEventData#setResourceClaims(Map)} instead.
      */
-    @Deprecated
+    @Deprecated()
```

Comment:
Is there a reason for adding `()` to this annotation?

Reply:
It's gone now - I had fixed it but just not pushed the branch.